client/js/admin.js

```diff
@@ -101,103 +184,26 @@ var attachHandles = function() {
 	})
 
 	$(".delete-path").click(function(e) {
-		var obj = $(this)
-		var isConfirming = obj.attr("confirming")
-		if (isConfirming == "false") {
+		e.stopPropagation()
+		const obj = $(this)
```

Comment:
Same disagreement with use of `const` here, I'm going to stop marking these since you get the idea.